tests/conformance/scripts/run-tests/runners/TestRunner.ts

```diff
@@ -0,0 +1,13 @@
+import { TestVariant } from "../../../variants/types";
+
+export abstract class ReadTestRunner {
```

Comment:
Interesting, why the choice of `abstract class` instead of `interface`? I haven't used abstract classes in TS before and they seem quite similar.

edit: oh, I see, it's so you can do `instanceof`.